ruby-training/trainees/alan/test_katas/03/JadenCase/jaden_case.rb

```diff
@@ -0,0 +1,16 @@
+
+
+class JadenCase
+    
+    def self.to_array_of_words (sentence)
+        sentence.split()
+    end
+
+    def self.capitalize_words! (sentence)
+        to_array_of_words(sentence).each { |word| word.capitalize!}
+    end
+
+    def self.make_it_in_jaden_case (sentence)
```

Comment:
This method name is a little too complicated for my taste... what is "it"? In cases like this, you can go with something that says the same amount of information in a single word such as `self.jadenify(sentence)`.